src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx

```diff
@@ -1,46 +1,147 @@
+      }),
+    [eventVenue, event]
+  );
+  const { isShown: isEventExpanded, toggle: toggleEventExpand } = useShowHide();
+  const { enterRoom } = useRoom({
+    room: eventRoom,
+    venueName: eventVenue?.name ?? "",
+  });
   const showDate = Boolean(
     differenceInCalendarDays(eventEndTime(event), eventStartTime(event))
   );
+  const isCurrentEventLive = isEventStartingSoon(event, 2 * EVENT_LIVE_RANGE);
+  const roomUrlParam = getUrlParamFromString(event.room ?? "");
+
+  const handleCopyEventLink = useCallback(
+    (e: React.MouseEvent<HTMLButtonElement>) => {
+      e.stopPropagation();
+
+      const eventLink = getFullVenueInsideUrl(roomUrlParam);
+      navigator.clipboard.writeText(eventLink);
+    },
+    [roomUrlParam]
+  );
+
+  const goToEventLocation = useCallback(() => {
+    if (!eventRoom) {
+      openUrl(roomUrlParam);
+
+      return;
+    }
+
+    if (event.room) {
+      enterRoom();
+    } else {
+      enterVenue(event.venueId);
+    }
+  }, [enterRoom, event, eventRoom, roomUrlParam]);
+
+  const infoContaier = classNames("ScheduleItemNG__info", {
+    "ScheduleItemNG__info--active": isCurrentEventLive,
+  });
+
+  const timeContainer = classNames("ScheduleItemNG__time", {
+    "ScheduleItemNG__time--live": isCurrentEventLive,
+  });
 
   return (
-    <div className="ScheduleItemNG">
-      <div className="ScheduleItemNG--info">
-        <span className="ScheduleItemNG--date">
-          {showDate &&
-            formatDateRelativeToNow(eventStartTime(event), formatDateOptions)}
+    <div className="ScheduleItemNG" onClick={toggleEventExpand}>
+      <div className={infoContaier}>
+        <span className="ScheduleItemNG__date">
+          {!isCurrentEventLive &&
+            showDate &&
+            formatDateRelativeToNow(eventStartTime(event))}
         </span>
 
-        <span className="ScheduleItemNG--time">
-          {formatTimeLocalised(eventStartTime(event))}
+        <span className={timeContainer}>
+          {isCurrentEventLive
+            ? "Live"
+            : formatTimeLocalised(eventStartTime(event))}
         </span>
 
-        <span className="ScheduleItemNG--date">
-          {showDate &&
-            formatDateRelativeToNow(eventEndTime(event), formatDateOptions)}
+        <span className="ScheduleItemNG__date ScheduleItemNG__time--end">
+          {showDate && formatDateRelativeToNow(eventEndTime(event))}
         </span>
 
-        <span className="ScheduleItemNG--time">
+        <span className="ScheduleItemNG__time ScheduleItemNG__time--end">
           {formatTimeLocalised(eventEndTime(event))}
         </span>
       </div>
-      <div className="ScheduleItemNG--name">{event.name}</div>
+      <div className="ScheduleItemNG__details">
+        <div className="ScheduleItemNG__name">{event.name}</div>
+        <div className="ScheduleItemNG__place">
+          <span className="ScheduleItemNG__place--location">in</span>{" "}
+          {roomUrlParam}
```

Comment:
please use the room name instead of this variable because right now it shows the venue id, when we want it to show the name of the room (like `event.room` ?)

Reply:
fixed in d1b21daee0ac88591d5d6eab07e0145e838b66a1